workers/unity/Packages/io.improbable.gdk.buildsystem/WorkerBuilder.cs

```diff
@@ -210,7 +206,7 @@ public static void Clean()
             {
                 currentContext = null;
                 PlayerSettings.iOS.sdkVersion = activeIOSSdkVersion;
-                PlayerSettings.SetScriptingBackend(buildTargetGroup, activeScriptingBackend);
+                PlayerSettings.SetScriptingBackend(buildTargetGroup, buildContext.ScriptingImplementation);
```

Comment:
This should swap back to the original backend after the build has finished, the flow being:

1. Save the scripting backend that we currently have.
2. Change the scripting backend.
3. Build
5. Swap the scripting backend back to that from (1) 

I believe the original implementation did this (with the `activeScriptingBackend`)

Reply:
Ohhh, right. I didn't realise that is what was happening. I thought it was just using an intermediate variable to store the backend for some reason